website/content/docs/api-gateway/upgrade-specific-versions.mdx

```diff
@@ -215,7 +369,27 @@ Ensure that the following requirements are met prior to upgrading:
   ```
 
   Repeat this step as needed until each of your cross-namespace routes have a corresponding `ReferencePolicy`.
-<a name="step-8"/>
+
+  Proceed with the [standard-upgrade](#standard-upgrade).
+
+
+## Standard Upgrade
+
+~> **Note:** When you see `VERSION` in examples of commands or configuration settings, replace `VERSION` with the version number of the release you are installing, like `0.2.0`. If there is a lower case "v" in front of `VERSION` the version number needs to follow the "v" as is `v0.2.0`
```

Comment:
1. See the comment I left RE: this callout on /consul-api-gateway-install.mdx
2. Shouldn't the standard upgrade be the first type on the page? 
3. These upgrade types are big enough that I probably would have put them on separate pages.

Reply:
I'm going to leave this page as is for right now. My current thinking is to move version-specific upgrade instructions to the Release Notes for that version. This page would then contain only the "standard" upgrade instructions with a big warning to review the Rel Notes before proceeding.